galasa-managers-parent/gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.jvmargs=-Xmx4096M
 
-jacocoEnabled= false
+jacocoEnabled= true
```

Comment:
I have changed this global setting to switch on jacoco, which may be controversial? 

I would like jacoco on regardless for the SDV manager due to the coverage threshold checking.

Reply:
I think that's a good idea too.